shared/engine/session.js

```diff
@@ -115,15 +115,15 @@ class Session {
       sessionID: this.id,
     }
 
-    rpcLog('engineInternal', 'session start call', this.id, method, this)
+    rpcLog('engineInternal', 'session start call', {id: this.id, method, this: this})
     const outgoingRequest = new OutgoingRequest(method, wrappedParam, wrappedCallback, this._makeWaitingHandler(true, method), this._invoke)
     this._outgoingRequests.push(outgoingRequest)
     outgoingRequest.send()
   }
 
   // We have an incoming call tied to a sessionID, called only by engine
   incomingCall (method: MethodKey, param: Object, response: ?Object): boolean {
-    rpcLog('engineInternal', 'session incoming call', this.id, method, this, response)
+    rpcLog('engineInternal', 'session incoming call', {id: this.id, method, this: this, response})
```

Comment:
will this work with chrome search? one reason i did it the original way was so the text was visible so you can command+f it